### DISCUSSION TITLE
PSA account blocked

### DISCUSSION THREAD

**Discussion by lvenditti** [flobz/psa_car_controller#351] [General]
Hi All,
after the installation of new version of psa-car-cantroller, my e-remote account seems blocked.
I can't access the e-remote functions nor from APP nor from psa-car-controller.
If I try to reinstall the APP on the phone when I try to register the phone the procedure stop when I insert the PIN:

procedure I follows:

1. install my citroen APP
2. insert credential (works fine)
3. APP asks to register the phone: recieved the SMS from Stellantis
4. Insert the SMS code
5. insert PIN (4 number) but at this point the APP do not goes head 

If I try to change my PIN I receive this error:

**2317: Forbidden operation (due to activated state/blocked state/upgradable state**

So now the My Citroen APP does not work and also the psa-car-controller is unable to access citroen, with this log:

_2022-04-17 19:43:58,763 :: INFO :: ["{'vin': 'VR7BCZKXxxxxxxx', 'vehicle_id': '1566c4933516b4e615331684454555526a4d544e6c4dxxxxxxxxxxxxxxx4e324d77597a67344f44453d', 'label': 'E-C4', 'brand': 'Citroën', 'abrp_name': 'citroen:ec4:21:50', 'battery_power': 53, 'fuel_capacity': 0, 'max_elec_consumption': 70, 'max_fuel_consumption': 0}"]
2022-04-17 19:43:59,833 :: ERROR :: can't refresh_remote_token: {'error': 'invalid_grant', 'error_description': 'grant is invalid', 'refresh_token': 'b553ba49-c2e0xxxxxxxxxxxxxxxxx'}
 Create a new one
2022-04-17 19:44:00,366 :: ERROR :: get_otp_code:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/psa_car_controller/psa/RemoteClient.py", line 193, in get_otp_code
    otp_code = self.otp.get_otp_code()
  File "/usr/local/lib/python3.7/dist-packages/psa_car_controller/psa/otp/otp.py", line 266, in get_otp_code
    if self.activation_start():
  File "/usr/local/lib/python3.7/dist-packages/psa_car_controller/psa/otp/otp.py", line 191, in activation_start
    raise ConfigException(xml)
psa_car_controller.psa.otp.otp.ConfigException: {'newversion': '2.0.0', 'newversionurl': 'http://m.inwebo.com/', 'err': 'NOK_BLOCKED'}
2022-04-17 19:44:00,374 :: ERROR :: Please redo otp config
2022-04-17 19:44:00,909 :: ERROR :: start_remote_control failed redo otp config

If I try to redo otp config but it fails.

Someone have any suggestion?

Is it possible that Citroen blocked my account? 
I read into my citroen app policy that they can block account if you use an external APP like psa-car-controller.

Bye

**Comment by dreimer1986** [comment]
In theory we have a FAQ for that, but in our case it is not working that way. (https://github.com/flobz/psa_car_controller/blob/master/FAQ.md)

Had the same problem yesterday, same hanging app all the same. Even called the german support and made them switch to panic mode by throwing too many tec terms ^^. My solution was to downgrade the app. I used for my case the myPeugeot app version 1.33 I got from APKMirror. Then it happily did what I expected it to. It wanted me to reset my PIN I entered wrong more than three times. (I did not, but who cares if it works then^^). Then it was working again.

**Reply by lvenditti** [reply]
Thanks for your reply. But also old version of my citroen App did not work.
It still to ask to register the phone but every attempt fails.
I opened a ticket to Citroën support from more than a week but no reply from them.

**Reply by dreimer1986** [reply]
The other thing we did, but seemed to have no real effect was remove the e-Remote Service and then re register it. Needs some time in the car sitting on active motor to sync though.

**Reply by dreimer1986** [reply]
You could try to type in the 4 digit pin wrong 5-6 times. My account resurrected with you have to change the PIN because entered wrong too often. (On old App version)

**Comment by lvenditti** [comment]
How you removed the subscription to e-remote?
From the APP?
from the citroen site?

Could you be so kind to share how you unsubscribed e-remote service?

Thanks for your help.
Ciao!

Il Ven 22 Apr 2022, 08:36 Daniel Reimer ***@***.***> ha
scritto:

**Reply by dreimer1986** [reply]
I can try to. I have the Peugeot app, but in theory it should bge similar. I have a site in my car settings which shows the active services and scrolling down to the very end you get a button which opens up a website to unsubscribe all services. As I said, it likely was not the reason my account worked again. Likely the invalidated pin was it. (And this very likely was the case because I tried to change it as last resort and was greeted with "2317: Forbidden operation (due to activated state/blocked state/upgradable state" and I tried it more than once.

![image](https://user-images.githubusercontent.com/29841368/164762028-f0c8ae93-a98d-465e-a72e-6657a5a4f34b.png)

**Reply by lvenditti** [reply]
Thank for reply. In my case App do not riquire to insert a PIN code it just want me to create a new one since no phone is registered and the PIN is no more defined (it was for sure).
But when i try to define it i receive an error message that ask me to rretry. Tried several time with same result. Still no answer from Citroën support.
:(

**Reply by lvenditti** [reply]
Ciao,
I removed subscription to e-remote service and then resubscribed again. Now the APP is working as expected (I am able to login and accept thi PIN) but last update still at 9 april 20222 15:51.
So reported SOC in non accurate and the car do not received remote control command (like programming or conditioning).

Hope that after some trips it will be updated.

**Reply by lvenditti** [reply]
after a couple of days also e-remote control is back!

Thanks so much for sharing your experience.

Ciao.

**Reply by dreimer1986** [reply]
Great :D Sad that thery had to over complicate things with their new app versions as it seems...
